src/components/common/Actions.vue

```diff
@@ -141,19 +141,6 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Lega
     this.setIsFilingPaying(true)
     let filingComplete
 
-    /** If it is a named company IA, validate NR before filing submission. This method is different
-     * from the processNameRequest method in App.vue. This method shows a generic message if
-     * the Name Request is not valid and clicking ok in the pop up redirects to the Manage Businesses
-     * dashboard */
-    if (this.isNamedBusiness) {
-      try {
-        await this.validateNameRequest(this.getNameRequestNumber)
```

Comment:
Don't we still need this if they are using a new NR?

Reply:
This will validate during that process of adding a new NR. Do you think they would need to Re-validate again when filing and paying?

Reply:
Not 100% sure.

On the one hand, I've heard that corrections are for staff only (for now) and will be done in one shot (no save and resume), so this reduces the chance of the NR no longer being valid during the session.

On the other hand, we do support save and resume, so it's possible that the user (staff or other) tries to file and pay on a different day than when they added the new NR, so this check could prevent data errors in that case.

Maybe this should be asked of Scott and/or BAs/POs...

Reply:
Ok, sounds good! I will inquire on this, as you mentioned above i feel like this is low enough risk to leave out at the moment.  
If it wasn't blocking Named Company filings that didn't include NR changes, i would 100% leave it in but in the meantime, it looks like a check we can do without at the moment.